### PR TITLE
(#374) updated predicate select/cards data/properties search

### PR DESCRIPTION
**Pull request by shlag3n**
#374 

* updated predicate select
image
* cards data
image
* properties search on the right

Note: to see predicates definitions new configuration profile should be created with [this updated predicates json](https://raw.githubusercontent.com/t3-innovation-network/desm/374-mapping-ui-usability/concepts/desmMappingPredicates.json)

### CODE REVIEW

File: app/javascript/components/align-and-fine-tune/stores/mappingStore.js [hunk 1]
```diff
@@ -69,14 +69,16 @@ export const noMatchPredicate = (predicate) =>
...
-  terms
-    .filter(
-      (term) =>
-        (options.pickSelected ? term.selected : !term.selected) &&
-        term.name.toLowerCase().includes(inputValue.toLowerCase())
-    )
+const filterTerms = (terms, inputValue, options = { pickSelected: false }) => {
+  const searchValue = inputValue.toLowerCase();
+  const searchBy = (term) =>
+    term.name.toLowerCase().includes(searchValue) ||
```
Comment (excelsior) [inline app/javascript/components/align-and-fine-tune/stores/mappingStore.js:75]: Looks like both term names and property comments can contain `NULL`s. Maybe those two should be coalesced to `""` unless it already happens somewhere in the serializer?

**Review by excelsior — APPROVED** [review]
Looks good!

File: app/javascript/components/align-and-fine-tune/stores/mappingStore.js (hunk 1 above)
Comment (shlag3n) [reply app/javascript/components/align-and-fine-tune/stores/mappingStore.js:75]: nice catch, will change to lodash `toLower`, name can't be blank, but other fields can. 
as about serializers we're using in-built one, thinking about adding `active_model_serializers` or `jsonapi-serializer`, any thoughts on this?

Comment (shlag3n) [reply app/javascript/components/align-and-fine-tune/stores/mappingStore.js:75]: @excelsior fix was force pushed

**Pull request merged**